float64[] matrix # 3x3 homography matrix
float32 scale
